feat: add config option to change response key

mycarrysun opened the pull request:
### Features
- Add a config option to allow the developer to change the response key that the debugger uses
  - This prevents naming conflicts with other libraries that also attach to the `debug` key in responses

Commit:
feat: add config option to change response key

mlanin commented on src/Debugger.php:
```diff
@@ -39,6 +39,8 @@ public function __construct(Storage $storage, Event $event)
         $this->storage = $storage;
         $this->event = $event;
 
+        $this->responseKey = config('api-debugger.response_key', 'debug');
```
Send this as a dependency through constructor, please. Or make a setter and overwrite in `boot()` method in ServiceProvider

mycarrysun replied:
I've added to `ServiceProvider::boot()`. Although I was not able to create a test for this in `DebuggerTest` because `ServiceProvider::boot()` has ran in the `setUp()` before the tests run. Any ideas how to test this feature?

mlanin replied:
There is no need to test service provider. Just add a test for `setResponseKey` and that will be enough.

Commit:
add const to Debugger class for default response key
init $responseKey as the default response key
moved setting the response key from config into the service provider

Commit:
make sure key is not empty

Commit:
feat: add getter for response key

Commit:
test: added test for setting response key

mycarrysun commented:
Thanks for being patient - test is now added! @mlanin

mlanin commented:
Thank you so much! This is awesome 👍
